src/umpire/strategy/QuickPool.hpp

```diff
@@ -95,27 +95,27 @@ class QuickPool : public AllocationStrategy, private mixins::AlignedAllocation {
     using size_type = std::size_t;
     using difference_type = std::ptrdiff_t;
 
-    pool_allocator() : pool{new util::FixedMallocPool{sizeof(Value)}}
+    pool_allocator() : pool{util::FixedMallocPool{sizeof(Value)}}
     {
     }
 
     /// BUG: Only required for MSVC
     template <typename U>
-    pool_allocator(const pool_allocator<U>& other) : pool{other.pool}
```

Comment:
This might be wrong... (the & might be needed actually)

Reply:
Yeah, it needs to be a reference.